ccmlib/common.py

```diff
@@ -670,7 +670,24 @@ def _get_scylla_version(install_dir):
             # 'i.j(.|-)dev[N]' < 'i.j.rc[N]' < 'i.j.k' < i.j(.|-)post[N]
             if re.fullmatch('(\d+!)?\d+([.-]\d+)*([a-z]+\d*)?([.-](post|dev|rc)\d*)*', v):
                 return v
-    return '3.0'
+
+    for scylla_exec in [
+        os.path.join(install_dir, 'scylla'),
+        os.path.join(install_dir, 'build', 'debug', 'scylla'),
+        os.path.join(install_dir, 'build', 'dev', 'scylla'),
+        os.path.join(install_dir, 'build', 'release', 'scylla'),
+        os.path.join(install_dir, 'bin', 'scylla'),
+        os.path.join(install_dir, 'scylla-core-package', 'bin', 'scylla'),
+    ]:
+        try:
+            scylla_version = subprocess.run(f"{scylla_exec} --version", shell=True, capture_output=True, text=True)
```

Comment:
That won't work when scylla is built in dbuild environement.
It must be installed first and patchelf'ed to point to the loadable libs dir.
Otherwise you get:
```
/home/bhalevy/dev/scylla-2/build/dev/scylla: error while loading shared libraries: libthrift-0.13.0.so: cannot open shared object file: No such file or directory
```